.editorconfig

```diff
@@ -0,0 +1,33 @@
+# top-most EditorConfig file
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.py]
+indent_size = 4
+
+[{.git*,Makefile}]
+indent_style = tab
+
+[{COMMIT_EDITMSG,MERGE_MSG,SQUASH_MSG}]
+max_line_length = 72
```

Comment:
I'd prefer if we use 99. (I know we don't have that yet other places). Thoughts? It seems much more do-able and makes it so you can fit a lot more into in-line comments without having to break them up.

Reply:
This is just for commit messages. I didn't set a max line length otherwise.

Reply:
clearly didn't read this close enough